docs/source/examples/mg-mode-part-ii.ipynb

```diff
@@ -0,0 +1,1661 @@
+    "Note that since this simulation included so few histories, it is reasonable to expect some data has not had any scores, and thus we could see division by zero errors.  This will show up as a runtime warning in the following step. The `Library` class is designed to gracefully handle these scenarios."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/nelsonag/git/openmc/openmc/tallies.py:1835: RuntimeWarning: invalid value encountered in true_divide\n",
+      "  self_rel_err = data['self']['std. dev.'] / data['self']['mean']\n",
+      "/home/nelsonag/git/openmc/openmc/tallies.py:1836: RuntimeWarning: invalid value encountered in true_divide\n",
+      "  other_rel_err = data['other']['std. dev.'] / data['other']['mean']\n",
+      "/home/nelsonag/git/openmc/openmc/tallies.py:1837: RuntimeWarning: invalid value encountered in true_divide\n",
+      "  new_tally._mean = data['self']['mean'] / data['other']['mean']\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Create a MGXS File which can then be written to disk\n",
+    "mgxs_file = mgxs_lib.create_mg_library(xs_type='macro', xsdata_names=['fuel', 'zircaloy', 'water'])\n",
+    "\n",
+    "# Write the file to disk using the default filename of \"mgxs.h5\"\n",
+    "mgxs_file.export_to_hdf5()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "OpenMC's multi-group mode uses the same input files as does the continuous-energy mode (materials, geometry, settings, plots, and tallies file).  Differences would include the use of a flag to tell the code to use multi-group transport, a location of the multi-group library file, and any changes needed in the materials.xml and geometry.xml files to re-define materials as necessary. The materials and geometry file changes could be necessary if materials or their nuclide/element/macroscopic constituents need to be renamed.\n",
+    "\n",
+    "In this example we have created macroscopic cross sections (by material), and thus we will need to change the material definitions accordingly.\n",
+    "\n",
+    "First we will create the new materials.xml file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# Instantiate our Macroscopic Data\n",
+    "fuel_macro = openmc.Macroscopic('fuel')\n",
+    "zircaloy_macro = openmc.Macroscopic('zircaloy')\n",
+    "water_macro = openmc.Macroscopic('water')\n",
+    "\n",
+    "# Now re-define our materials to use the Multi-Group macroscopic data\n",
+    "# instead of the continuous-energy data.\n",
+    "# 1.6 enriched fuel UO2\n",
+    "fuel_mg = openmc.Material(name='UO2')\n",
+    "fuel_mg.add_macroscopic(fuel_macro)\n",
```

Comment:
Again, you don't really need to create a `Macroscopic` object above. Would be shorter to just pass the string here